transformer_engine/common/__init__.py

```diff
@@ -3,39 +3,3 @@
 # See LICENSE for license information.
 
 """FW agnostic user-end APIs"""
-import ctypes
-import os
-import platform
-import subprocess
-import sys
-
-
-def get_te_path():
-    """Find Transformer Engine install path using pip"""
-
-    command = [sys.executable, "-m", "pip", "show", "transformer_engine"]
-    result = subprocess.run(command, capture_output=True, check=True, text=True)
-    result = result.stdout.replace("\n", ":").split(":")
-    return result[result.index("Location") + 1].strip()
-
-
-def _load_library():
-    """Load shared library with Transformer Engine C extensions"""
-
-    system = platform.system()
-    if system == "Linux":
-        extension = "so"
-    elif system == "Darwin":
-        extension = "dylib"
-    elif system == "Windows":
-        extension = "dll"
-    else:
-        raise RuntimeError(f"Unsupported operating system ({system})")
-    lib_name = "libtransformer_engine." + extension
-    dll_path = get_te_path()
-    dll_path = os.path.join(dll_path, lib_name)
-
-    return ctypes.CDLL(dll_path, mode=ctypes.RTLD_GLOBAL)
-
-
-_TE_LIB_CTYPES = _load_library()
```

Comment:
This is just really nice. Handling rpaths correctly is so much better than manually loading libraries.

Reply:
I'm still sad https://github.com/NVIDIA/TransformerEngine/pull/858 didn't get merged.